apps/prairielearn/src/components/PageLayout.tsx

```diff
@@ -277,7 +285,10 @@ export function PageLayout({
         ${sideNavEnabled ? compiledScriptTag('pageLayoutClient.ts') : ''}
       </head>
       <body
-        class="${resolvedOptions.fullHeight ? 'd-flex flex-column h-100' : ''}"
+        class="${clsx(
+          resolvedOptions.fullHeight && 'd-flex flex-column h-100',
```

Comment:
does clsx dedupe?

Reply:
What problem do you anticipate with duplicates?

Reply:
no problems, just slightly ugly html

Reply:
Fair enough! Changed in 1b265d5.